components/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/handlers/security/saml2/SAML2AuthenticationHandler.java

```diff
@@ -649,7 +649,7 @@ private VerbInfoDTO getVerbInfoForApp(String webAppContext, String webAppVersion
      * @return true if JWT is enabled in App Manager config.
      */
     private boolean isJWTEnabled(){
-        return AppMDAO.jwtGenerator != null;
+        return AppMDAO.isJWTEnabled();
```

Comment:
Can move isJWTEnabled() to AppManagerConfiguration and use it via 
AppManagerConfiguration config = ServiceReferenceHolder.getInstance().getAPIManagerConfiguration();